### DISCUSSION TITLE
Export/Import  assemble file

### DISCUSSION THREAD

**Discussion by taschenlampe** [Dvlv/BoxBuddyRS#133] [General]
Hi!

I have a distrobox that I have create within BB.
 I have installed several applications, I can see them in the UI of BB. I wonder how to export this list of apps (as a assemble.ini I guess), so I can re-create a distrobox within BB using this assemble.ini.

Is there any chance BB could do that?

Cheers!

**Comment by Dvlv** [comment]
Hello,
I can't see this functionality in distrobox, so it won't be added to BoxBuddy.

If anyone else knows of a distrobox command which does this, please let me know.

**Reply by taschenlampe** [reply]
Hey!

Thanks for your feedback. I started an discussion @[distrobox'](https://github.com/89luca89/distrobox/discussions/1601) git page. Maybe someone could help me creating this assemble.ini that I would like to have.

Cheers! You're doing great with BB—keep it up! 😊
